php-classes/Slate/CBL/Calculators/Growth/PerformanceLevelMinusBaseline.php

```diff
@@ -8,7 +8,13 @@ class PerformanceLevelMinusBaseline implements IGrowthCalculator
 {
     public static function calculateGrowth(StudentCompetency $StudentCompetency)
     {
-        $baseline = $StudentCompetency->BaselineRating;
+        $baseline = round($StudentCompetency->BaselineRating, 1);
+        $performanceLevel = round($StudentCompetency->getDemonstrationsAverage(), 1);
+
+        // baseline & performance level must > 0
+        if (!$performanceLevel || !$baseline) {
+            return false;
```

Comment:
@nbey shouldn't this be `null`?

Reply:
Currently the method returns `false` when there is no growth to calculate. The `StudentCompetency` class handles returning null, but caches the `false` value.